include/deal.II/grid/connectivity.h

```diff
@@ -166,7 +166,7 @@ namespace internal
       unsigned int
       n_entities(const unsigned int d) const override
       {
-        static std::array<unsigned int, 3> table = {2, 1};
+        static std::array<unsigned int, 3> table = {{2, 1}};
```

Comment:
Unrelated to this pull request, I find omitting the last zero here (and in similar places) rather confusing.

Reply:
This was a mistake. The vector should actually have the length 2. I have fixed it in #10742.